tests/cases/fourslash/referencesForStringLiteralPropertyNames6.ts

```diff
@@ -2,7 +2,7 @@
 
 ////const x = function () { return 111111; }
 ////x.[|{| "isWriteAccess": true, "isDefinition": true |}someProperty|] = 5;
-////x["[|someProperty|]"] = 3;
+////x["[|{| "isWriteAccess": true, "isDefinition": true |}someProperty|]"] = 3;
```

Comment:
Worked on same tests in my PR and have question about this one: why this string will be definition if this "static" property was defined above?

Reply:
Yeah, you’re right. This is wrong.

Reply:
Actually, I’m not sure. If I change this line to property access (`x.someProperty = 3`), it still wants `isDefinition: true`. It might be wrong, but it appears to be consistently wrong, and probably nothing to do with my changes here...

Reply:
Yeah, after fixing a problem with go-to-definition that turned out to be unrelated to this, I’m satisfied with calling this unrelated to this PR.